tests/test_ci_gpu_smoke_contract.py

```
@@ -0,0 +1,102 @@
+"""Repository-owned CI contracts for deterministic GPU smoke provisioning."""
+
+from __future__ import annotations
+
+from pathlib import Path
+
+
+_CI_WORKFLOW = Path(__file__).parents[1] / ".github" / "workflows" / "ci.yml"
+_APT_SOURCES = "/tmp/fast-mlsirm-ubuntu.sources"
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/fast-mlsirm-ubuntu.sources"

(S108)